KinKalGeom/src/CRV.cc

```diff
@@ -0,0 +1,34 @@
+#include "Offline/KinKalGeom/inc/CRV.hh"
+namespace mu2e {
+  namespace KinKalGeom {
+    using KinKal::VEC3;
+    using KinKal::Rectangle;
+    // currently use hard-coded geometry
+    CRV::CRV() :
+      r1_{ std::make_shared<Rectangle>(VEC3(-1.0,0.0,0.0),VEC3(0.0,0.0,1.0), VEC3(-2537.28,275.53,-10281.1),2070,2275)},
```

Comment:
both r(right?) and l(left?) have plane normal in the -x direction. I think it would be better to have the plane normal point away from the detector origin.